tasks/copy.js

```diff
@@ -92,35 +92,35 @@ gulp.task('copy:fonts', () => {
  */
 gulp.task('copy:assets', callback => {
 
-    const ending = 'package.json';
-    const getPkg = filepath => {
-        const path = filepath.slice(0, -ending.length); // the parent directory that contains the package.json
+    const getPkg = path => {
```

Comment:
Nit: In the method below you use the term "package" and here you're using "pkg", I'd stick with using "package" as it's a little clearer.